ts/store/reducers/wallet/outcomeCode.ts

```diff
@@ -0,0 +1,137 @@
+import { Option } from "fp-ts/lib/Option";
+import * as pot from "italia-ts-commons/lib/pot";
+import { getType } from "typesafe-actions";
+import I18n from "../../../i18n";
+import {
+  OutcomeCode,
+  OutcomeCodes,
+  OutcomeCodesKey
+} from "../../../types/outcomeCode";
+import { Action } from "../../actions/types";
+import { outcomeCodeRetrieved } from "../../actions/wallet/outcomeCode";
+import authorizationDenied from "../../../../img/servicesStatus/error-detail-icon.png";
+import genericError from "../../../../img/wallet/errors/generic-error-icon.png";
+import sessionExpired from "../../../../img/wallet/errors/payment-expired-icon.png";
+import cardProblemOrOperationCanceled from "../../../../img/wallet/errors/payment-unknown-icon.png";
+import { GlobalState } from "../types";
+
+export type OutcomeCodeState = {
+  outcomeCode: pot.Pot<OutcomeCode, Error>;
```

Comment:
Since you are not using nothing from pot, `except` some and `none`, could it better to shape this section as an Option?

`outcomeCode: Option<OutcomeCode>;`

Reply:
Make sense! Modified in [c9aa70b](https://github.com/pagopa/io-app/pull/2842/commits/c9aa70bdfccd113d8276b22128ecad4677cec3ef)